### PR TITLE
Bump Navigation Version

### PR DESCRIPTION
**Pull request by svanherk**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
       "d2l-table": "1.7.5",
       "d2l-tabs": "0.4.5",
       "d2l-telemetry-browser-client": "0.4.0",
-      "d2l-tooltip": "2.0.7",
```
Comment (svanherk) [inline bower.json]: Fixing animation, @danyakarras is this good to go in?

File: bower.json [hunk 2]
```diff
@@ -30,17 +30,17 @@
...
-      "d2l-my-courses": "6.0.9",
+      "d2l-my-courses": "6.0.10",
       "d2l-my-dashboards": "1.0.6",
-      "d2l-navigation": "2.1.0",
+      "d2l-navigation": "2.1.3",
       "d2l-offscreen": "3.0.6",
       "d2l-opt-in-flyout-webcomponent": "1.2.1",
       "d2l-organization-hm-behavior": "2.0.5",
       "d2l-organizations": "0.1.14-hybrid",
-      "d2l-outcomes-level-of-achievements": "1.0.13",
```
Comment (svanherk) [inline bower.json]: Just Polymer 3 prep (fixing linting)

File: bower.json (hunk 1 above)
Comment (dbatiste) [reply bower.json]: I think Danya might not be in till a little later (time difference in all).  From what I know of this change, should be good to go!

File: bower.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
       "d2l-colors": "3.1.3",
       "d2l-course-image": "2.1.6",
       "d2l-dnd-sortable": "0.0.4",
-      "d2l-dropdown": "6.6.4",
+      "d2l-dropdown": "6.6.5",
```
Comment (dbatiste) [inline bower.json]: This one is expected too.  Thanks!

**Pull request merged**